core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1855,6 +1857,28 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Because we can only have one active Spark Context per JVM and there are times when multiple
+   * applications may wish to share a SparkContext, this function may be used to get or instantiate
+   * a SparkContext and register it as a singleton object.
+   * 
+   * Note: This function cannot be used to create multiple spark contexts even if multiple contexts
+   * are allowed. Multiple contexts will still need to be created with explicit calls to the 
+   * SparkContext constructor. 
+   */
+  def getOrCreate(config: SparkConf): SparkContext = {
+    if (activeContext.get() != null) {
+      setActiveContext(new SparkContext(config), 
+        config.getBoolean("spark.driver.allowMultipleContexts", false))
```

Comment:
I believe this can just be `false`, because if we create a new `SparkContext` here then it already means that there is not already an existing one.